src/com/spikes2212/genericsubsystems/LimitedSubsystem.java

```diff
@@ -5,25 +5,56 @@
 
 public abstract class LimitedSubsystem extends Subsystem {
 	private SpeedController motor;
-
+	
+	/**
+	 * Constructs a subsystem using a single SpeedController
```

Comment:
this is not your average subsystem, this is a LIMITED SUBSYSTEM! (woah! itsocool!)